tests/test-root.php

```
@@ -1,10 +1,175 @@
+				'iri'  => 'http://example.com/',
+			),
+			array(
+				'type' => 'invalid',
+				'iri'  => 'not-a-url',
+			),
+		);
+
+		$_POST['_wpnonce'] = wp_create_nonce( 'odg-schema' );
+
+		$schema = new ODG_Admin_Schema();
+		$result = $schema->save_schema();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$result'. (undefined)

(UnusedLocalVariable)